### DISCUSSION TITLE
[Question]: What is the default username and password?

### DISCUSSION THREAD

**Discussion by Kreijstal** [danny-avila/LibreChat#3837] [Q&A] [answered]
I installed librechat via docker but how do I log in?

**Comment by fuegovic** [comment, marked as answer]
There's no default, you need to sign up
![image](https://github.com/user-attachments/assets/2c5dde6a-affe-4db0-8d7a-5d0dba86720e)

**Reply by Kreijstal** [reply]
sign up to my own instance? lmao ok

**Reply by fuegovic** [reply]
the first account you create will be attributed the `ADMIN` role
